### ISSUE TITLE
Hubot does no respond

### ISSUE THREAD

**Issue by kitop** [unixcharles/hubot-talker#4]
I'm running the talker adapter on Heroku, and Hubot is not answering to any command. I've tried:
`@hubot help`
`Hubot help`
`hubot help`
and didn't get any response.

Procfile:

```
web: bin/hubot -a talker -n Hubot
```

packaje.json

``` json
{
  "name":        "hosted-hubot",
  "version":     "2.1.4",
  "author":      "GitHub Inc.",
  "keywords":    "github hubot campfire bot",
  "description": "A simple helpful Robot for your Company",
  "licenses":     [{
    "type":         "MIT",
    "url":          "http://github.com/github/hubot/raw/master/LICENSE"
  }],

  "repository" : {
    "type": "git",
    "url":  "http://github.com/github/hubot.git"
  },

  "dependencies": {
    "hubot-talker":       ">= 1.0.1",
    "hubot":              "2.1.4",
    "hubot-scripts":      "2.0.6",
    "optparse":           "1.0.3",
    "wolfram":            "0.1.1",
    "soupselect":         "0.2.0",
    "htmlparser":         "1.7.5",
    "jsdom":              "0.2.13",
    "underscore":         "1.3.0",
    "underscore.string":  "2.1.1"
  }
}
```

Hubot is connected to the rooms, and doing `heroku logs` I can see it's receiving the msgs and actions.

**Comment by kitop** [comment]
Also, on doing a GET to /hubot/version, the reponse is 
`Cannot GET /hubot/version`

It was working before switching (from campfire to talker)

**Comment by kitop** [comment]
This was fixed on Pull Request #5, so it can be closed
